IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayFacilityPartTypeRailwayCrossing/DocPropertySet.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayFacilityPartTypeRailwayCrossing" UniqueId="7f547f91-6dd0-161c-9824-1f608dd660d9" ApplicableType="IfcFacilityPart" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="IsAccessibleByVehicle_3R5N45JWj9UumSu_fUKeWz" />
+		<DocProperty xsi:nil="true" href="ContractID_3rU499iXbFohf4SGYszyGc" />
+		<DocProperty xsi:nil="true" href="HasDrainage_30Z3LZrU15p9CBV6oSf4at" />
+		<DocProperty xsi:nil="true" href="IsPrivateOwner_1E7ZEm8I13sg3M3fKSTa3V" />
+		<DocProperty xsi:nil="true" href="PermissibleRoadLoad_0dZw4UYSDFVPrQibCUZruL" />
```

Comment:
PermissiblePavementLoad --> it might not be a road??

Reply:
Updated https://github.com/bSI-InfraRoom/IFC-Specification/pull/164/commits/ab85f083c2717d30ef42c6a47dcce352a5b16e89